scripts/update-ui-tests.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+# Script for updating the UI tests for a new rust stable version.
+# Exit on error
+set -e
+
+# by default current rust stable will be used
+RUSTUP_RUN=""
+# check if we have a parameter
+# ./substrate/.maintain/update-rust-stable.sh 1.70
+if [ "$#" -e 1 ]; then
```

Comment:
This check isn't quite sufficient: command-bot calls this script with an "" argument if no rust version was specified, in which case $# is 1 but $1 is "".

Reply:
Oh and also there's a typo here, the equality operator is `-eq`, not `-e` :)

Reply:
yea, actually -z "$1" is  what i had to use :) fixed